src/NServiceBus.Core.Tests/API/APIApprovals.ApproveNServiceBus.approved.txt

```diff
@@ -1843,6 +1844,14 @@ namespace NServiceBus.Features
     {
         protected internal override void Setup(NServiceBus.Features.FeatureConfigurationContext context) { }
     }
+    public class RoutingComponent
```

Comment:
So everything that goes onto the feature setup context is "components" or do you envision a contex.GetComponent<Routing>() if there is more? (transport, pipeline etc)